### DISCUSSION TITLE
Keine Werte, System-Wartung nicht aufrufbar

### DISCUSSION THREAD

**Discussion by totti4ever** [OpenCCU/OpenCCU#2089] [🇩🇪 Fragen&Antworten]
### Describe the issue you are experiencing

Seit gestern Abend gibts in der CCU-UI keine Werte mehr und somit auch nicht in HASS. Zusätzlich kann ich die System-Wartung auch nicht mehr aufrufen, alles kriegt einen grauen Overlay, aber es kommt kein modales Fenster. Die anderen Knöpfe unter Einstellungen funktionieren.

Im Docker Log ist nichts zu sehen, das ist aber auch nicht sehr detailliert.

### Describe the behavior you expected

Werte werden wie zuvor angezeigt, Dialog "Systemwartung" öffnet sich

### Steps to reproduce the issue

1. Web UI öffnen
2. Keine Werte da
3. System-Wartung unter Einstellungen öffnen

### What is the version this bug report is based on?

3.65.11.20221005

### Which base platform are you running?

rpi4 (RaspberryPi4)

### Which HomeMatic/homematicIP radio module are you using?

HmIP-RFUSB

### Anything in the logs that might be useful for us?

```txt
Docker Image: ghcr.io/jens-maus/raspberrymatic:latest
```

**Comment by jens-maus** [comment]
Alles kein Bug in RaspberryMatic, daher kein Issue Ticket sondern ein Diskussionssbeitrag mit bitte um Hilfe für das lokale Problem. Ergo, verschoben in den Diskussionsbereich.

**Reply by totti4ever** [reply]
Kann ich denn auf das Log auch ohne UI zugreifen? Das wäre sehr hilfreich könnte ich mir vorstellen

**Reply by totti4ever** [reply]
@jens-maus, wie kann es eigentlich kein Bug sein, wenn ich die Zentralenwartung nicht mehr öffnen kann??

**Reply by jens-maus** [reply]
Weil es 40k andere Nutzer inkl. mir können. Das wird ein Folgefehler sein weil irgendwas in deiner lokalen Installation durcheinandergekommen ist. Und aus den Informationen die du geliefert hast kann man auch nichts anderes erkennen das auf einen allgemeinen Bug schließen lässt.
